tools/upgrade-deps/bin/cli.js

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env node
+/* eslint-disable import/extensions */
```

Comment:
It came from the nodejs resolution. From what I have understood it can works without extension if I rename all files with .esm extension. 

So here is the trade off: I would like to keep .js file in our code base as from what I have seen, esm and cjs files may create issue with editors and tooling.
For example jest do not see the file if not .js.

https://nodejs.org/api/esm.html#esm_mandatory_file_extensions